### ISSUE TITLE
I have a issue when integrate the this gem. 

### ISSUE THREAD

**Issue by nexondevs** [solidusio-contrib/solidus_globalize#152]
Hello Team,
I am interested in setting up an e-commerce store using Solidus. In this store, I would like to have dynamic and status data that can be translated into different languages such as Arabic, Urdu, and English.

I have attempted to install the `solidus_globalize gem`. While I was able to install it successfully, I encountered an issue. When I access the admin dashboard and navigated to the product page, I clicked on the translate option and tried to select another locale like` :arb `for Arabic. However, the product fields were not visible for translation, unlike the` default :en` field.

If anyone has experience with integrating this gem, could you please share the GitHub link or any relevant resources? I would greatly appreciate it.

If there are any articles or videos available on this topic, kindly share them with me. Thank you in advance for your assistance.

solidus version: 4

Videos:

https://github.com/solidusio-contrib/solidus_globalize/assets/69003744/383a75e3-cf21-43f5-8dfb-b086185053cd

**Comment by jarednorman** [comment]
The ISO code for Arabic is "ar", not "arb", though I do not think that is causing your issue as Portuguese wasn't working either.

Are there any errors in the browser console?

**Comment by adnanmirza1** [comment]
@jarednorman Yes, I am experiencing the issue on my console. I have also attached a screenshot for reference. 
`Uncaught TypeError: Cannot read properties of undefined (reading 'replace')`

**Comment by jarednorman** [comment]
I would assume the issue is with one of these calls to `replace`: https://github.com/solidusio-contrib/solidus_globalize/blob/8e4c497168353526ec8bd94ea727a9d288d9ac9b/app/assets/javascripts/spree/backend/solidus_globalize.js#L13-L15
